PSPDFKitPlugin/PSPDFKitPlugin.m

```diff
@@ -1747,4 +1754,38 @@ - (void)exportXFDF:(CDVInvokedUrlCommand *)command {
     }
 }
 
+- (void)processAnnotations:(CDVInvokedUrlCommand *)command {
+    PSPDFAnnotationChange change = (PSPDFAnnotationChange)[self optionsValueForKeys:@[[command argumentAtIndex:0]] ofType:@"PSPDFAnnotationChange" withDefault:PSPDFAnnotationChangeEmbed];
+    NSURL *processedDocumentURL = [self writableFileURLWithPath:[command argumentAtIndex:1] override:YES copyIfNeeded:NO];
+
+    // The annotation type is optional. We default to `All` if it's not specified.
+    NSString *typeString = [command argumentAtIndex:2] ?: [command argumentAtIndex:3];
```

Comment:
When moving the callback to the last parameter, this could be simplified.

```suggestion
    NSString *typeString = [command argumentAtIndex:2];
```

Reply:
Even if the type would be omitted, and the callback is the parameter at index 2, the check below would still make sure we use all annotation types if it was omitted.